CONTRIBUTING.md

```diff
@@ -100,6 +100,24 @@ $ git push fork feature
 
 Open a pull request against the main `opentelemetry-python` repo.
 
+Pull requests are also tested for their compatibility with packages distributed 
+by OpenTelemetry in the [OpenTelemetry Python Contrib Repository](https://github.com/open-telemetry/opentelemetry-python.git).
+
+If a pull request (PR) introduces a change that would break the compatibility of
+these packages with the Core packages in this repo, a separate PR should be
+opened in the Contrib repo with changes to make the packages compatible.
+
+Follow these steps:
+1. Open Core repo PR (Contrib Tests will fail)
+2. Open Contrib repo PR and modify its `CORE_REPO_SHA` in `.github/workflows/test.yml`
+to equal the commit SHA of the Core repo PR to pass tests
+3. Modify the Core repo PR `CONTRIB_REPO_SHA` in `.github/workflows/test.yml` to
+equal the commit SHA of the Contrib repo PR to pass Contrib repo tests (a sanity
+check for the Maintainers & Approvers)
```

Comment:
Optionally you can _not_ do this, and just wait for the Contrib PR to get merged. The Contrib PR getting merged proves this Core PR is good!

Once Contrib PR is merged, since this points to master, you can just rerun tests and see it all work.